### ISSUE TITLE
[CC] Replication Star has an out-of-place 0 Lumen requirement

### ISSUE THREAD

**Issue by Shahelion** [ReikaKalseki/Reika_Mods_Issues#1210]
So, I have some Pylon Casting recipes in my lexicon, although I don't have the right combinations of Lumens for most of them, so I hadn't unlocked the progression for pylon casting crafting yet. However, I went and made a Replication Star, which is in the previous tier, and it unlocked the End Game tier. Looking closer, the Replication Star has an additional page in the recipe after the Item Casting Stands page, the Lumens requirements page, but it doesn't actually require any. No colors, and the amount is 0. This caused early access to End Game tier. This doesn't seem intentional.

Sorry for the probably confusing giant paragraph.

TLDR: Replication Star isn't Pylon Casting, why does it have an empty Lumen requirement? Early End-Game tier trigger.

![2016-11-10_22 04 50](https://cloud.githubusercontent.com/assets/6917535/20221929/38b000ee-a7e8-11e6-82cd-ccc578bfc0ac.png)

**Comment by ReikaKalseki** [comment]
Now it is.
